### PR TITLE
Fix: Enter key updates plan

### PR DESCRIPTION
**Pull request by RehkMansa**
# Fixes 
When users press the enter key, it updates the plan as well as onclick of the update button. To test, create a plan, click on the submit button or press the enter key, plan would be updated

### CODE REVIEW

File: src/components/screens/Plan/PlanSidebar.tsx [hunk 1]
```diff
@@ -64,7 +65,9 @@ export const PlanSidebar = (props: PlanSidebarProps) => {
 		setFrequency(props.plan?.frequency ?? 'som');
 	}, [props.plan?.frequency]);
 
-	const submitPlanSettingsChange = () => {
+	const submitPlanSettingsChange = (e: React.SyntheticEvent) => {
+		e.preventDefault();
+		console.log('submitPlanSettingsChange');
```
Comment (Andreasgdp) [inline src/components/screens/Plan/PlanSidebar.tsx:70]: Remove console.log 😉

**Review by Andreasgdp — CHANGES_REQUESTED** [review]
It looks good, if you just remove the console.log then it is good to go 👍

File: src/components/screens/Plan/PlanSidebar.tsx (hunk 1 above)
Comment (RehkMansa) [reply src/components/screens/Plan/PlanSidebar.tsx]: Rookie Mistake from my end, hahaha, I've removed that, thanks

Comment (Andreasgdp) [reply src/components/screens/Plan/PlanSidebar.tsx]: No problem, it happens

**Review by Andreasgdp — APPROVED** [review]

**Pull request merged**